lambdas/service/app.py

```diff
@@ -471,19 +471,27 @@ def openapi():
 }
 
 fast_health_keys = {
-    **{
-        prop.key: format_description(prop.description)
-        for prop in HealthController.fast_properties['service']
-    },
-    **health_up_key
+    p: d for p, d in sorted(
+        {
+            **{
+                prop.key: format_description(prop.description)
+                for prop in HealthController.fast_properties['service']
+            },
+            **health_up_key
+        }.items(),
+        key=lambda key: key[0])
 }
 
 health_all_keys = {
-    **{
-        prop.key: format_description(prop.description)
-        for prop in HealthController.all_properties
-    },
-    **health_up_key
+    p: d for p, d in sorted(
+        {
+            **{
+                prop.key: format_description(prop.description)
+                for prop in HealthController.all_properties
```

Comment:
For `HealthController.all_properties`, I think the definition could be change to a tuple safely too, and would avoid the need for these changes. If it can't be done safely, then I think it would still make sense to sort at the definition instead of here.

Reply:
https://github.com/DataBiosphere/azul/pull/2997#discussion_r625339700